icgc-legacy-sftp/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: ICGC Legacy SFTP - Pensato chart with mounted volume as file system
```

Comment:
Update comment, this version has no stateful set (no mounted volume or bucket). This relies on configuring an external file or s3 system